client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/test/unified-design-picker.tsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import nock from 'nock';
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import * as wpcomProxyRequest from 'wpcom-proxy-request';
+import wpcomXhrRequest from 'wpcom-xhr-request';
+import {
```

Comment:
[super nit, very subjective] I think we can just inline the mock data in this test. It's easier to read as we don't have to jump into multiple files. I don't see why we need to separate them into separate files.

```
jest.mock( '../../../../../hooks/use-site', () => ( {
	useSite: () => ( { ID: 1} ),
} ) );
```

```
nock( 'https://public-api.wordpress.com' )
	.persist()
	.get( '/wpcom/v2/starter-designs' )
	.query( true )
	.reply( 200, {
		generated: {
			designs: [],
		},
		static: {
			designs: [],
		},
	} );
```

etc

Reply:
hmmm, I'm not sure what's the best practice but I'd prefer to separate the mock data from the test file as much as possible so that
1. The test file could focus on the test cases instead of the data. 
2. When we jump to a single test file, we won't see lots of mock data at the beginning
3. The mock data could be re-used 

Additionally, we could also mock the API response globally and we don't need to establish API mocking in a single test file every time.

I know everyone has their preference but I don't assume it's a big issue to jump into multiple files. For example, when you develop a new feature or work on a bug, you still need to jump into multiple files 😂

Reply:
All right. Sure thing!